configure.sh

```diff
@@ -0,0 +1,463 @@
+#!/bin/sh
+#--------------------------------------------------------------------------#
+
+usage () {
+cat <<EOF
+Usage: $0 [<build type>] [<option> ...]
+
+Build types:
+  production
+  debug
+  testing
+  competition
+
+
+General options;
+  -h, --help               display this help and exit
+  --prefix=STR             install directory
+  --build-prefix=STR       prefix build directory with given prefix
```

Comment:
Seems to be GitHub again. In vim it's aligned.

Reply:
Gaaaaah, I had the same issue recently in one of my PRs. It drove me nuts because I was trying to find some accidental tabs but it turned out that it is just GitHub screwing up :/ Sorry about that.